### DISCUSSION TITLE
走査 → スキャン に統一

### DISCUSSION THREAD

**Discussion by noborus** [pgsql-jp/jpug-doc#3327] [NGとする語]
### 重要度

中

### 問題点

scanの訳としてスキャンが多数派ですが、 「走査」としている場合もあります。
スキャンの方が一般に使用されているので統一して良いと思います。

### 背景

「sequential scan」の訳後がシーケンシャルスキャン, シーケンシャル走査, 順スキャン, 順次スキャン, 逐次スキャンで統一されていないですが、全体的に「scan」が「スキャン」と「走査」で統一されていない問題がありました。

### 解決方法

「走査」を「スキャン」に置き換え。
その後、「順スキャン」,「 順次スキャン」,「 逐次スキャン」を「シーケンシャルスキャン」に置き換える。

### 注意点

「スキャン」に置き換えても不自然なところがないか、一応確認。

### 貢献者として記載可否

以前の記載名を使用

**Comment by KenichiroTanaka** [comment]
$find . -name "*.sgml"|xargs grep "走査"
$
完了したと思いますのでクローズします。

**Comment by noborus** [comment]
NG語に「走査」を追加します
